test/shared/src/test/scala/zio/test/TestAspectSpec.scala

```diff
@@ -132,6 +143,32 @@ object TestAspectSpec extends DefaultRuntime {
       failed(spec)
     }
 
+  def timeoutMakesTestsFailAfterGivenDuration: Future[Boolean] =
```

Comment:
These tests are a little awkward to write because we would really like to use the `MockClock` here but we don't want to be using ZIO Test to test ZIO Test. I would shorten the timeout durations because we are using real time here and I don't think the actual amount of time matters since the effects will never terminate. You could probably just use 1 nanosecond here and 1 and 2 nanoseconds below.